app/Resources/views/Project/edit.html.twig

```diff
@@ -1,8 +1,17 @@
 {% extends '::layout.html.twig' %}
 
-{% block page_title -%} Project Edit {% endblock %}
+    {% block edit %}
+        {{ form(edit_form) }}
+    {% endblock%}
+<ul class="record_actions">
+    <li>
+        <a href="{{ path('project') }}">
+            Back to the list
+        </a>
+    </li>
+    <li>{{ form(delete_form) }}</li>
+</ul>
 
-{% block content %}
```

Comment:
@isaias1503 Why are you removing the block content? where is the open tags? check that this layout isn't broke